src/api/java/appeng/api/features/IRecipeHandlerRegistry.java

```diff
@@ -65,11 +67,13 @@
 	 * @return A recipe handler by name, returns null on failure.
 	 */
 	@Nullable
+	@Deprecated
 	ICraftHandler getCraftHandlerFor( String name );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1133)